src/ExtLibevLoop.php

```diff
@@ -110,6 +112,7 @@ public function removeWriteStream($stream)
 
         if (isset($this->writeEvents[$key])) {
             $this->writeEvents[$key]->stop();
+            $this->loop->remove($this->readEvents[$key]);
```

Comment:
Looks like this line causes build failures in PHP < 7, eg. https://travis-ci.org/reactphp/event-loop/jobs/337955988#L1463

Reply:
Thanks for spotting, fixed the copy-pasta 🍝  :shipit: